### PR TITLE
test: add coverage for Unread action in MarkChapterUseCase

### PR DESCRIPTION
**Pull request by nonproto**
💡 What: Added unit test `MarkChapterUseCaseTest` for the `ChapterMarkActions.Unread` edge case, verifying that the pages left and last read parameters are passed correctly to the `updateChaptersProgress` database mapped chapters.

🎯 Why: Ensure the correct progress parameters are passed down through `MarkChapterUseCase` to the database, preventing regressions when managing Unread chapter status.

---
*PR created automatically by Jules for task [1030392476708131236](https://jules.google.com/task/1030392476708131236) started by @nonproto*

### CODE REVIEW

File: app/src/test/java/org/nekomanga/usecases/chapters/MarkChapterUseCaseTest.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            coVerify(exactly = 1) { mockPreparedPut.executeOnIO() }
+
+            val updatedChapters = capturedChapters.captured
+            assertEquals(1, updatedChapters.size)
+
+            val updatedChapter = updatedChapters.first()
+            assertEquals(1L, updatedChapter.id)
+            assertFalse("Chapter should be marked as unread", updatedChapter.read)
+            assertEquals(expectedLastRead, updatedChapter.last_page_read)
+            assertEquals(expectedPagesLeft, updatedChapter.pages_left)
```
[the remark this answers is not in the document]
Comment (nonproto) [reply app/src/test/java/org/nekomanga/usecases/chapters/MarkChapterUseCaseTest.kt:88]: > medium
> 
> For consistency with the project's dependencies, which include `kotest.assertions`, it's better to use Kotest's assertion functions instead of JUnit's `Assert`. This improves readability and maintains a uniform testing style.
> 
> Please also add `import io.kotest.matchers.shouldBe` and remove the imports for `org.junit.Assert`.
> 
> ```suggestion
>             updatedChapters.size shouldBe 1
> 
>             val updatedChapter = updatedChapters.first()
>             updatedChapter.id shouldBe 1L
>             updatedChapter.read shouldBe false
>             updatedChapter.last_page_read shouldBe expectedLastRead
>             updatedChapter.pages_left shouldBe expectedPagesLeft
> ```

@jules
